decidim-elections/app/cells/decidim/votings/voting_map_cell.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    class VotingMapCell < VotingMCell
+      include Decidim::MapHelper
+      include Decidim::Votings::MapHelper
+
+      delegate :snippets, to: :controller
+
+      def show
+        # return unless Decidim::Map.available?(:geocoding, :dynamic)
```

Comment:
What to do with this comment? @entantoencuanto

Reply:
My bad, debugging purposes, I'll restore it